soyjs/exec_test.go

```diff
@@ -1045,6 +1047,60 @@ soy.$$escapeHtml = function(arg) { return arg; };
 	}
 }
 
+func TestES6(t *testing.T) {
+	bundle := soy.NewBundle()
+	bundle.AddTemplateString("test_formatter.soy", `
+	{namespace test}
```

Comment:
Multiline strings shouldn't generally be indented, because the indent characters become part of the content e.g. 

```
	AddTemplateString("foo.soy", `{namespace test}
{template .foo}
 lakdsfjd
{/template}`)

```